dawn/src/dawn4py/serialization/utils.py

```diff
@@ -718,7 +718,8 @@ def make_literal_access_expr(value: str, type: BuiltinType.TypeID) -> LiteralAcc
     :param type:    Builtin type id of the literal.
     """
     builtin_type = BuiltinType()
-    builtin_type.type_id = type
+    if builtin_type.type_id != type:
```

Comment:
Is this change necessary?

Reply:
Yes, it throws the second error in the description otherwise.

Reply:
Ah ok, an error gets raised in the = operation otherwise. Thanks for clarifying!